lib/languages.ts

```diff
@@ -917,6 +917,18 @@ const definitions: Record<LanguageKey, LanguageDefinition> = {
         previewFilter: null,
         monacoDisassembly: null,
     },
+    sway: {
+        name: 'sway',
+        monaco: 'rust', // use rust for syntax highlighting
```

Comment:
Is rust a good choice here? Are there any major differences? It's _fairly_ straightforward to extend an existing language's syntax highlighter.

Reply:
Good point, just added a sway definition so we can use that instead of rusts.